src/ucp/proto/proto_init.c

```diff
@@ -267,8 +267,13 @@ ucp_proto_init_parallel_stages(const ucp_proto_common_init_params_t *params,
     ucs_array_for_each(elem, &concave) {
         range             = &caps->ranges[caps->num_ranges];
         range->max_length = elem->max_length;
-        range->node       = ucp_proto_perf_node_new_data(params->super.proto_name,
+        if (fabs(bias) > UCP_PROTO_PERF_EPSILON) {
+            range->node   = ucp_proto_perf_node_new_data(params->super.proto_name,
+                                                         "bias %f", bias);
```

Comment:
Maybe to many digits after the dot, `%.3f` ?

Reply:
There may be cases then when we have note about `0.000` bias since `UCP_PROTO_PERF_EPSILON` is `1e-15`.